tests/ecmp/test_ecmp_sai_value.py

```diff
@@ -0,0 +1,129 @@
+import pytest
+import re
+import logging
+from tests.common import config_reload
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.platform.processes_utils import wait_critical_processes
+
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.asic('broadcom'),
+    pytest.mark.topology('t1'),
+    pytest.mark.disable_loganalyzer
+]
+
+seed_cmd = [
+    'bcmcmd "getreg RTAG7_HASH_SEED_A"',
```

Comment:
should we organize this per ASIC family, TD2 maybe different with TH/TH2

Reply:
Will check the differences.